### ISSUE TITLE
RT integration

### ISSUE THREAD

**Issue by harunurhan** [inspirehep/record-editor#146]
## UI

- List of tickets (get open tickets which for `record.control_number == ticket.RecordID`)
 - each ticket: date, queue name, link to RT system.
 - each ticket has modal preview with title and description.
 - each can be closed

- Create a new ticket.
 - a modal
   - Queue name, Owner dropdowns are dynamic.
   - Subject, Status, Description to fill manually
   - Description could be generated from a template based on the queue name

## Services needed

- get all tickets by `RecordID`
- close a ticket
- get RT users
- get Queue names
- create a ticket

Blocked by https://github.com/inspirehep/inspire-next/issues/2418

**Comment by StellaCh** [comment]
## rt ticket:
<img width="946" alt="new"  src="https://user-images.githubusercontent.com/24955201/27373133-5d7a6a1c-5667-11e7-8a2a-e8dd7a7e34ce.png">

- On hover over date/time: show popover with ticket title and description
- On mouse click on date/time: redirect to the ticket in rt
- Resolve ticket when clicking on glyphicon glyphicon-ok-sign

## Modal for new ticket:
<img width="407" alt="screen shot 2017-06-21 at 09 58 40" src="https://user-images.githubusercontent.com/24955201/27373296-0da1b6ac-5668-11e7-987d-9db3f9711106.png">
Changes:

- No status dropdown anymore
- Template content appears depending on the queue
- Only relevant queues appear in the list
## Multiple tickets (rare case):
<img width="835" alt="screen shot 2017-06-21 at 10 01 45" src="https://user-images.githubusercontent.com/24955201/27373431-8510b670-5668-11e7-9d46-18b52659fa28.png">

**Comment by StellaCh** [comment]
- To make the glyphicon glyphicon-ok-sign look more like a button, you can use some css for borders and shadows like in https://designmodo.com/3d-css3-button/ 
```
.button {
    display: inline-block;
    text-decoration: none;
    color: #fff;
    font-weight: bold;
    background-color: #538fbe;
    padding: 20px 70px;
    font-size: 24px;
    border: 1px solid #2d6898;
     
    -webkit-border-radius: 5px;
    -moz-border-radius: 5px;
    border-radius: 5px;
    text-shadow: 0px -1px 0px rgba(0,0,0,.5);
    -webkit-box-shadow: 0px 6px 0px #2b638f, 0px 3px 15px rgba(0,0,0,.4), inset 0px 1px 0px rgba(255,255,255,.3), inset 0px 0px 3px rgba(255,255,255,.5);
    -moz-box-shadow: 0px 6px 0px #2b638f, 0px 3px 15px rgba(0,0,0,.4), inset 0px 1px 0px rgba(255,255,255,.3), inset 0px 0px 3px rgba(255,255,255,.5);
    box-shadow: 0px 6px 0px #2b638f, 0px 3px 15px rgba(0,0,0,.4), inset 0px 1px 0px rgba(255,255,255,.3), inset 0px 0px 3px rgba(255,255,255,.5);  
}
```
- To make the queues look less like a button, make the height smaller and style them similar to the keyword tags we have in qa: 
![keywords](https://user-images.githubusercontent.com/24955201/28357209-ec418816-6c6a-11e7-91c4-351839dac81b.PNG)

**Comment by harunurhan** [comment]
fixed
